horace_core/sqw/@Experiment/Experiment.m

```diff
@@ -360,17 +372,17 @@
                 nspe(i) = exp_cellarray{i}.n_runs;
             end
             n_tot = sum(nspe);
-            instr  = repmat({IX_null_inst()},1,n_tot );
-            sampl  = repmat({IX_null_sample()},1,n_tot);
+            instr  = cell(1,n_tot);
+            sampl  = cell(1,n_tot);
```

Comment:
fully refactored. Can not even find this place any more. I believe, your favorite `null_stuff` was still there -- just fills cellarray, instead of converting arrays to cellarrays.